data/advanced_preferences.cfg

```diff
@@ -21,6 +21,13 @@
     [/option]
 [/advanced_preference]
 
+[advanced_preference]
+    field=addon_icons
+    name= _ "Whether to show icons in the add-ons list on the add-ons manager"
```

Comment:
I think it would be clearer to say that the icons are not even sent be the server and that that can save some bandwidth so it's clear what the option is for.

Reply:
How about:
> Disabling saves bandwidth by not sending icons for the add-ons list in the add-ons manager

Reply:
Updated.